projects/kit/components/calendar-range/calendar-range.component.ts

```diff
@@ -174,6 +180,7 @@ export class TuiCalendarRangeComponent implements TuiWithOptionalMinMax<TuiDay>
 
     updateValue(value: TuiDayRange | null) {
         this.value = value;
```

Comment:
we try to change `@Input` property here. Maybe lets change `@Input() value` to setter?

Reply:
No, this is a two-way binding sync, it's common practice to update internal property and emit new value so the external is synced.